interface/RooSimultaneousOpt.h

```diff
@@ -26,9 +26,9 @@ class RooSimultaneousOpt : public RooSimultaneous {
         _extraConstraints("extraConstraints", this, other._extraConstraints),
         _channelMasks("channelMasks", this, other._channelMasks)  {}
 
-  virtual RooSimultaneousOpt *clone(const char* name=0) const { return new RooSimultaneousOpt(*this, name); }
+  RooSimultaneousOpt *clone(const char* name=0) const override { return new RooSimultaneousOpt(*this, name); }
 
-  virtual ~RooSimultaneousOpt() ;
+  ~RooSimultaneousOpt() override ;
 
   virtual RooAbsReal* createNLL(RooAbsData& data, const RooLinkedList& cmdList) ;
```

Comment:
From the compilation output with ROOT 6.22 it looks like this method should also have an override. 
I guess it was not automatically added because you ran clang-tidy with a ROOT version > 6.22. Could you please check?

Reply:
Indeed, but that depends on the root version. I will fix this in a separate commit/PR:
https://github.com/cms-analysis/HiggsAnalysis-CombinedLimit/pull/949

You want me to add that commit also in this PR? Would that make things easier for you?

Reply:
for this particular method I think it is fine to add it in another PR. 
But I just wonder how to make sure that we stay consistent and backward compatible with these override statements. For now it would be great if you could check with 6.22 and 6.26 if it is not difficult, thanks!

Reply:
I have combined the PRs after all in this one, that made it easier for me to validate.

I compiled this PR now with 6.22, 6.26, and 6.30 to check that there are no cases in which is get warnings related to overrides. And it is not the case!